internal/cloudsql/instance.go

```diff
@@ -117,6 +121,7 @@ func NewInstance(instance string, client *sqladmin.Service, key *rsa.PrivateKey,
 	}
 	i := &Instance{
 		connName:       cn,
+		clientLimiter:  rate.NewLimiter(rate.Every(30*time.Second), 2),
```

Comment:
What's the significance of `2` here? Without it, the rate limiter allows a single call every 30 seconds. With it, clients could at most make two calls every 30 seconds. Is that right?

Reply:
It's a leaky bucket limiter, with a capacity of 2. So essentially you can "burst" two requests immediately, but will never exceed an average rate of 1 per 30s outside of that. It means that if a one off error occurs, we can instantly try again. However if we consistently have errors, we won't exceed the 1/30s number. 

https://en.wikipedia.org/wiki/Leaky_bucket